content/applications/finance/accounting/taxes/avatax/avatax_use.rst

```diff
@@ -100,3 +100,43 @@ value. To recalculate the taxes, click :guilabel:`Compute taxes using Avatax`.
 
 .. tip::
    Tax calculation can even be done on negative subtotals and credit notes.
+
+Logging
+=======
+
+It is possible to log Avalara/*AvaTax* actions in Odoo for further analysis or verification of
+functionality. Logging is accessible through the *AvaTax* settings.
+
+To start logging *AvaTax* actions, first, navigate to the :menuselection:`Accounting app -->
+Configuration --> Settings`. Then, in the :guilabel:`Taxes` section, under the :guilabel:`AvaTax`
+settings, click :guilabel:`Start logging for 30 minutes`. Upon starting the log process, Odoo will
+log all Avalara/*AvaTax* actions performed in the database.
+
+To view the logs, click on :guilabel:`Show logs` to the right of the :guilabel:`Start logging for 30
+minutes` button. This will reveal a detailed list of Avalara/*AvaTax* actions. This list is sortable
+by the following columns:
+
+- :guilabel:`Created on`
+- :guilabel:`Created by`
+- :guilabel:`Database name`
+- :guilabel:`Type`
+- :guilabel:`Name`
+- :guilabel:`Level`
+- :guilabel:`Path`
+- :guilabel:`Line`
+- :guilabel:`Function`
+
+.. tip::
+   Custom fields can be made using Odoo *Studio*. Click the :icon:`fa-ellipsis-v`
+   :guilabel:`ellipsis` menu to the far right of the header row. Then click :icon:`fa-plus`
+   :guilabel:`Add custom field`. This action will open Odoo *Studio*.
+
+   Odoo *Studio* requires a *custom* pricing plan. Consult the database's customer success manager,
+   for more information on switching plans, or to see if Odoo *Studio* is including in the
+   database's current pricing plan.
```

Comment:
I think this deserves an important admonition separate from and directly following the tip admonition